spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/core/subscriber/PubSubSubscriberTemplate.java

```diff
@@ -118,6 +133,8 @@ public Subscriber subscribe(String subscription, MessageReceiver messageReceiver
 
 	@Override
 	public PubsubMessage pullNext(String subscription) {
+		Assert.hasText(subscription, "The subscription can't be null or empty.");
```

Comment:
May we move this assert into the `pullAndAck()` which we delegate anyway?

Reply:
Fixed.  `pullAndAck` has the assert.